fluent/migrate/transforms.py

```diff
@@ -253,23 +254,34 @@ def __call__(self, ctx):
             variant, = variants
             return evaluate(ctx, self.foreach(variant))
 
-        last_index = min(len(variants), len(keys)) - 1
+        # The default cldr form should be the first we have in DEFAULTS
```

Comment:
nit: CLDR

I'm confused by this comment though: aren't we taking the last from DEFAULTS?

Reply:
I had refactored this, and forgot to adjust the comment. Sorry, pushed a follow-up.